src/pages/Condition.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import ContentViewer from "../components/ContentViewer";
+import HeartCounter from "../components/HeartCounter";
+import Button from "../components/Button";
+import LineGraph from "../components/graphs/LineGraph";
+import RadarGraph from "../components/graphs/RadarGraph";
+import { sortData } from "../utils/graphData";
+import { getCondition } from "../api/condition";
+
+const ConditionWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  color: ${({ theme }) => theme.text.main};
+
+  .graph {
+    width: 700px;
+  }
+
+  .graph button {
+    float: right;
+  }
+
+  .status {
+    flex-basis: 680px;
+    align-items: center;
+  }
+`;
+
+const StatusInfo = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 0.8fr 1fr);
+  justify-items: start;
+  column-gap: 1rem;
+  padding: 20px;
+  width: 630px;
+  margin: 0 auto;
+  border-radius: 20px;
+  box-shadow: ${({ theme }) => theme.shadow.main};
+  background-color: ${({ theme }) => theme.background.main};
+  font-size: ${({ theme }) => theme.fontSizes.small};
+`;
+
+function Condition() {
+  const [isRadarGraph, setIsRadarGraph] = useState(true);
+  const [categories, setCategories] = useState([]);
+  const [dataPerDate, setDataPerDate] = useState(null);
+  const [status, setStatus] = useState([]);
+  const [heartCount, setHeartCount] = useState(0);
+
+  useEffect(() => {
+    async function loadCondition() {
+      const conditionData = await getCondition();
```

Comment:
```suggestion
      const conditionData = await getCondition();
      
      if (!conditionData) {
        return;
      }
```
에러가 발생한 경우, 모달을 띄워주는 동안 아래의 로직이 실행되지 않도록 막아주어야 할 것 같습니다.